pkg/cmd/events_list.go

```diff
@@ -76,10 +76,6 @@ var validEvents = map[string]bool{
 	"issuing_card.updated":                         true,
 	"issuing_cardholder.created":                   true,
 	"issuing_cardholder.updated":                   true,
-	"issuing_dispute.created":                      true,
-	"issuing_dispute.updated":                      true,
-	"issuing_settlement.created":                   true,
-	"issuing_settlement.updated":                   true,
```

Comment:
Were these supposed to get removed? :|

Reply:
Yep, that's expected, they shouldn't have been made public in the first place.